back-end/helpers/post/transcript_data_formatter.js

```diff
@@ -9,6 +9,7 @@ export default class TranscriptFormatter {
                     prev = this.#updateContent(entry, content, prev);
                 }
                 else{
+                    this.#addChild(content, prev, "[END OF CONVERSATION]", content[prev].project_id);
```

Comment:
```suggestion
                    this.#addChild(content, prev, "User Ended Conversation.", content[prev].project_id);
```
Might be a lil nicer